app/Index.tsx

```diff
@@ -1,12 +1,13 @@
 /// <reference path="../typings/tsd.d.ts" />
```

Comment:
Don't checkin change to this file I guess. Otherwise all of you will have conflict.